bin/elasticsearchFiles/startES.sh

```diff
@@ -0,0 +1,24 @@
+###script to start elasticsearch server###
+
+logs=~/ELK_POC_HOME/logs 
```

Comment:
Please replace all the ~/ELK_POC_HOME with env variable $ELK_POC_HOME 
e.g logs= $ELK_POC_HOME/logs.

Add a comment in the readme/document, that it is expected to set these ENV variables by the user.